api/python/cellxgene_census/src/cellxgene_census/experimental/pp/_online.py

```diff
@@ -74,12 +92,21 @@ def __init__(self, n_samples: int, n_variables: int):
 
         self.u = np.zeros(n_variables, dtype=np.float64)
         self.n_samples = n_samples
+        # If we want to exclude zeros, we need to keep track of the denominator
+        self.exclude_zeros = exclude_zeros
+        self.denominator = np.zeros(n_variables)
```

Comment:
You have introduced a second naming scheme for the sample count (`n`), calling it `denominator`.  The existing code uses `n` which is the convention in stats.  See the code in `MeanVarianceAccumulator`.

Recommend being consistent, as it will make the code easier to understand.